rules/macos/persistence_modification_sublime_app_plugin_or_script.toml

```diff
@@ -37,9 +37,10 @@ file where event.type in ("change", "creation") and file.extension : "py" and
     (
       "/Applications/Sublime Text*.app/Contents/MacOS/Sublime Text*",
       "/usr/local/Cellar/git/*/bin/git",
+      "/Library/Developer/CommandLineTools/usr/bin/git",
       "/usr/libexec/xpcproxy",
       "/System/Library/PrivateFrameworks/DesktopServicesPriv.framework/Versions/A/Resources/DesktopServicesHelper",
-      "/Applications/Sublime Text.app/Contents/MacOS/plugin_host"
+      "/Applications/Sublime Text.app/Contents/MacOS/plugin_host*"
```

Comment:
instead of adding this line, just modify L38 to be

```diff
-      "/Applications/Sublime Text*.app/Contents/MacOS/Sublime Text*",
+      "/Applications/Sublime Text*.app/Contents/*",
```